R/internals.R

```diff
@@ -95,25 +93,36 @@
     gsub("__", "_", .)
 
   names(stat_df) <- new_names
-  stat_df <- stat_df[-1,]
+  stat_df <- stat_df[-1, ]
 
   stat_df <- stat_df %>% dplyr::select(-.data[["Matches"]])
 
   remove_rows <- min(grep("Season", stat_df$Season)):nrow(stat_df)
 
   stat_df <- stat_df[-remove_rows, ]
 
-  if(any(grepl("LgRank", names(stat_df)))){
-    cols_to_transform <- stat_df %>%
-      dplyr::select(-.data[["Season"]], -.data[["Squad"]], -.data[["Country"]], -.data[["Comp"]], -.data[["LgRank"]]) %>% names()
-  } else {
-    cols_to_transform <- stat_df %>%
-      dplyr::select(-.data[["Season"]], -.data[["Squad"]], -.data[["Country"]], -.data[["Comp"]]) %>% names()
+  cols_to_transform <- stat_df %>%
```

Comment:
@francescozonaro Have you tested other existing functions where this cleaning code was used to ensure it doesn't cause problems with those existing functions?